metrics/network/remote_network/README.md

```diff
@@ -34,3 +38,19 @@ $ cd metrics/network/remote_network
 $ bash remote-networking-iperf3.sh -b -i eno1 -u tester -a 10.xxx.xxx.xxx
 
 ```
+
+## Running the `remote-networking-memory-smem.sh` test
+
+`remote-networking-memory-smem.sh` should run in host A. The script needs the following inputs to run:
+- Arguments of the test to run. In this case the argument is `-r` for RSS memory.
+- Interface name where swarm will run.
+- User of the host B.
+- IP address of the host B.
+
+The `remote-networking-memory-smem.sh` test may be run manually:
+
+```
+$ cd metrics/network/remote_network
```

Comment:
I think normally we invoke the tests from the metrics directory, and not the subdirs themselves don't we? Can you check if this invocation example is uniform with any others we list etc.
And just to check - the 'save metrics' when invoking in this dir works correctly does it? (I forget...)

Reply:
@grahamwhaley this is using the same example like https://github.com/clearcontainers/tests/tree/master/metrics/network